src/averageAges.js

```diff
@@ -15,11 +15,22 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+  const ageOfDeath = men.map(person => person.died - person.born);
+  const centureResultArr = men.filter(person =>
+    Math.ceil(person.died / 100) === century
+  );
```

Comment:
combine men and centureResultArr filters into one
then you don't need to use map twice for both ageOfDeath and diedThisCentury, you will already have ONE array of needed men